File: packages/clipboard/ios/Sources/OukekClipboard/OukekClipboard.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

CAP_PLUGIN(OukekClipboard, "OukekClipboard",
    CAP_PLUGIN_METHOD(getChangeCount, CAPPluginReturnPromise);
    CAP_PLUGIN_METHOD(read, CAPPluginReturnPromise);
    CAP_PLUGIN_METHOD(write, CAPPluginReturnPromise);
    CAP_PLUGIN_METHOD(clear, CAPPluginReturnPromise);
)

File: script/pluginTpl/ios/Sources/OukekClipboard/OukekClipboard.swift
import Foundation
import Capacitor
import UIKit

/**
 * OukekClipboard for handling clipboard operations
 */
@objc(OukekClipboard)
public class OukekClipboard: CAPPlugin {
    
    override public func load() {
        // 插件加载时的初始化
        NotificationCenter.default.addObserver(self,
                                            selector: #selector(handleAppLaunch),
                                            name: UIApplication.didFinishLaunchingNotification,
                                            object: nil)
    }
}